recipes/marshmallow-sqlalchemy/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "marshmallow-sqlalchemy" %}
+{% set version = "0.12.0" %}
+{% set sha256 = "8a7e56dcf95a27287359812e3285ed72d333f72a50d8ca760fb1c1527ea2c52b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - marshmallow >=2.0.0
+    - sqlalchemy >=0.9.7
```

Comment:
Please drop anything from `build` that is not strictly a requirement at build time. Probably just need `python` and `setuptools`.